### ISSUE TITLE
Using bulleted or numbered list causes blank CKEditor

### ISSUE THREAD

**Issue by eddiedale** [craftcms/ckeditor#127]
### Description

When adding bulleted list or numbered list to the config, the whole field dissapears.
The issue happened after upgrading from Craft 4.4.14 and CKEditor 3.4.0 to Craft 4.5.2 and CKEditor 3.5.0

Error from Console:
```
[Error] Unhandled Promise Rejection: CKEditorError: document-list-feature-conflict {"conflictPlugin":"ListEditing"}
Read more: https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-document-list-feature-conflict
	dispatchException (ckeditor5-craftcms.js:2:723310)
	(anonym funksjon) (ckeditor5-craftcms.js:2:720327)
	Vb (ckeditor5-craftcms.js:2:725056)
	l (ckeditor5-craftcms.js:2:732576)
[Error] TypeError: undefined is not an object (evaluating 'i.options.bubble')
	triggerShortcut (garnish.js:2:94352)
	triggerShortcut
	dispatch (jquery.js:2:43398)
[Error] TypeError: undefined is not an object (evaluating 'i.options.bubble')
	triggerShortcut (garnish.js:2:94352)
	triggerShortcut
	dispatch (jquery.js:2:43398)
```
https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-document-list-feature-conflict

### Steps to reproduce

1. Have a basic field without list items
<img width="368" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/5be6dfd0-f7d9-4bda-b1d9-e1ab79d84b0a">

2. Add a "bulleted list" to the field in the config

<img width="402" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/a1c75528-cd96-4618-874c-31de2d6d37ef">

3. See the field dissapear

<img width="195" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/8fb506fd-f84a-4fac-9842-51f08cb2725c">

### Additional info

- Craft version: Craft Pro 4.5.2
- PHP version: 8.2.0
- Database driver & version: MariaDB 10.4.27
- Plugins & versions:  CKEditor 3.5.0

**Comment by brandonkelly** [comment]
I’m not able to reproduce that. What is your “List Plugin” setting set to? And do you have any Config Options entered? If so, please paste the whole Config Options value.

**Comment by eddiedale** [comment]
@brandonkelly Ah, ok, so the problem is rather that I don't see any "List Plugin" setting.

I have in the meantime created a different new Craft CMS Project, and there I saw the "List Plugin" settings, which was new to me.

On the project for this Github issue everything is up to date (plugins and Craft), but none of my 4 configs have the "List Plugin" visible.

<img width="1726" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/4030591a-431f-404d-b2bc-470581e405c3">

Example of one:
```
return {
    link: {
        decorators: {
            openInNewTab: {
                mode: 'manual',
                label: 'Åpne i ny tab',
                defaultValue: true,
                attributes: {
                    target: '_blank',
                    rel: 'noopener noreferrer'
                }
          }
        }
    }
}
```

I also just tried to create a new one, and it doesn't show there either:
<img width="2306" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/9ba5dd59-7050-49c2-9ead-4ac794868374">


The console log shows the following:
```
widget-toolbar-no-items
TypeError: undefined is not an object (evaluating 'i.options.bubble')
```

<img width="978" alt="image" src="https://github.com/craftcms/ckeditor/assets/995836/a4880f9b-0621-4adf-b64b-88ac266f4eb6">

## Development specs
```
DDEV describe
craftcms PHP8.2
nginx-fpm
docroot:'web'
NodeJS:16

Browser: Safari 16.6
```

The issue came after updating the project:
Craft 4.4.14 -> Craft 4.5.2 
CKEditor 3.4.0 -> CKEditor 3.5.0

**Comment by brandonkelly** [comment]
That JS error was just fixed in Craft 4.5.3 (https://github.com/craftcms/cms/issues/13605), but shouldn’t be causing any actual downstream issues.

It seems like CKEditor 3.5.0 didn’t get fully installed. Can you try nuking the `vendor` folder and running `composer update`, and see if that helps?

**Comment by eddiedale** [comment]
Ok, so I just nuked the `vendor` folder and ran `composer update` without luck.

But thereafter I updated to Craft 4.5.3 and CKEditor 3.5.1. This solved the issue.
So it seems like it actually caused downstream issues, but I don't know. 🤷‍♂️ 

At least it works now with the update! Thanks!
